components/navigation/NavBar.js

```diff
@@ -71,10 +82,9 @@ export default function NavBar({
     router.push(href);
   };
 
-  // To be enabled on the last PR
-  // useEffect(() => {
-  //   changeLanguage(browserLanguageDetector(), false);
-  // }, []);
+  useEffect(() => {
+    changeLanguage(browserLanguageDetector(), false);
+  }, []);
```

Comment:
What function does this useEffect do on initial render? specify this in a comment.

Reply:
done